packages/runner/injection/main.js

```diff
@@ -16,6 +17,8 @@ if (!Cypress) {
 Cypress in the parent window but it is missing. This should never happen and likely is a bug. Please open an issue.')
 }
 
+patchXmlHttpRequest(window)
```

Comment:
@mschile why do we need to patch here? Is this needed for intercept or something similar?